packages/binding.core/src/let.ts

```diff
@@ -3,8 +3,10 @@ import {
     applyBindingsToDescendants
 } from '@tko/bind'
 
+import type { BindingContext } from '@tko/bind'
+
 export default {
-  init: function (element, valueAccessor, allBindings, viewModel, bindingContext) {
+  init: function (element, valueAccessor, allBindings, viewModel, bindingContext: BindingContext) { // allBindings and viewModel not used!
```

Comment:
We could use the convention to start unused vars with `_` e.g.

```ts
  init: function (element, valueAccessor, _allBindings, _viewModel, bindingContext: BindingContext) {
```

then the comment isn't needed and in many editors the variable will have a different color for the unused var

Reply:
@M-Kirchhoff has this adapted.

Actually, we should collect such things in a general TKO style guide.